history/characters/11000_dark_iron.txt

```diff
@@ -2622,7 +2622,7 @@ coren_direbrew={
 	}
 	355.1.1={
 		religion = fire_cult
-		effect={ set_variable = { name = wc_perks_needed_for_level_1_physical_trait_value } }
+		effect={ set_variable = { name = wc_endurance_physical_lifestyle_additional_perks_variable value = wc_endurance_physical_lifestyle_additional_perks_variable value = wc_endurance_physical_lifestyle_additional_perks_variable value = wc_perks_needed_for_level_1_physical_trait_value } }
```

Comment:
You should check `error.log` after fixes

> [18:51:29][jomini_eventtarget.cpp:676]: Failed to find a valid event target link 'wc_endurance_physical_lifestyle_additional_perks_variable' at  file: history/characters/11000_dark_iron.txt line: 2625
> [18:51:29][jomini_eventtarget.cpp:676]: Failed to find a valid event target link 'wc_endurance_physical_lifestyle_additional_perks_variable' at  file: history/characters/11000_dark_iron.txt line: 2625
> [18:51:29][jomini_eventtarget.cpp:676]: Failed to find a valid event target link 'wc_endurance_physical_lifestyle_additional_perks_variable' at  file: history/characters/11000_dark_iron.txt line: 2657
> [18:51:29][jomini_eventtarget.cpp:676]: Failed to find a valid event target link 'wc_endurance_physical_lifestyle_additional_perks_variable' at  file: history/characters/11000_dark_iron.txt line: 2680
> 

Remove the extra values in all three of them, should fix it